server/server.py

```diff
@@ -64,13 +66,15 @@
                                          "/slack/events", app)
 
 
+@csrf.exempt
 @app.route('/')
```

Comment:
This endpoint doesn't need to be exempted, right? It'll only be accessed directly through a browser, not through redirects from Slack/Github/etc.

Reply:
I can't seem to reach the endpoint if it isn't exempted cause it still seems to attempt verifying a csrf token. After doing a bit more research I'm getting the vibe that protection against csrf attacks are more for applications that rely on forms and keep track of sessions, while currently we just provide a REST application. Might need another opinion on this too, but even then I don't think the root endpoint does anything to expose a security vulnerability.

Reply:
Yeah, that's true. In that case, perhaps we just shouldn't bother with this, then?

Reply:
Maybe, unless we decide that we want to interact with rocket through forms later on, but I'd imagine that wouldn't be in the near future.

Reply:
True - that's very much not in the plans right now (though if we ever add @cheukyin699's idea for an admin console...)